### PR TITLE
Layout inheritance exercise

### LINKED ISSUE #21: Exercise: layout inheritance
in `06-templates.md`

need an episode on creating new layouts based on old layouts

### PR DESCRIPTION
**Pull request by tobyhodges**
Fixes #21

### CODE REVIEW

File: _episodes/07-templates.md [hunk 1]
```diff
@@ -17,6 +17,41 @@ keypoints:
 - "You can avoid duplicated effort by basing new layouts on previous ones"
 ---
 
+> ## Expanding the layout
+>
+> We will probably want to include the contact line we added in the previous section
+> in every standard page on our site.
+> It would be better placed into the layout template,
```
Comment (sstevens2) [inline _episodes/07-templates.md]:
```suggestion
> Instead, we can place it in the layout template
```
Slight language change suggestion

File: _episodes/07-templates.md [hunk 2]
```diff
@@ -17,6 +17,41 @@ keypoints:
...
+> Check that this works by applying that layout to the `team.md` page.
+>
+> > ## Solution
+> > Your new layout file, `_layouts/page-contact.html`,
+> > should look like this:
+> >
+> > ~~~
+> > ---
+> > layout: page
+> > ---
```
Comment (sstevens2) [inline _episodes/07-templates.md:39]:
```suggestion
> > ---
> > 
```
Sometimes issues with rendering when there is no blank line after the header.

**Review by sstevens2 — APPROVED** [review]
Looks good to me!  I included two small suggestions.

File: _episodes/07-templates.md (hunk 2 above)
Comment (tobyhodges) [reply _episodes/07-templates.md:39]: well spotted, thanks!

**Pull request merged**